src/variables/utils/buildVarsOption.ts

```diff
@@ -32,7 +30,8 @@ export const buildVarsOption = (variables: VariableAssignment[]): File => ({
   type: 'File',
   package: null,
   imports: null,
-  body: [
+  // only claim namespace `v` if needed
+  ...(!variables.length ? {body: []} : {body: [
```

Comment:
We have two choices:
1. only send the extern when optional variables are present. It means having changes be correct in >=5 places across the codebase.
2. always create the extern. But only claims the namespace `v` if variables are provided via extern.